librz/core/panels.c

```diff
@@ -3518,7 +3518,7 @@ int __function_cb(void *user) {
 
 int __symbols_cb(void *user) {
 	RzCore *core = (RzCore *)user;
-	rz_core_cmdf(core, "aa");
+	rz_core_analysis_all (core);
```

Comment:
this needs to be tested a bit. DId you already tried to see if it works well. I have some doubts just because it seems the `aa` command in `cmd_analysis.c` does a lot more than just this function.

Reply:
>I have some doubts just because it seems the aa command in 
> cmd_analysis.c does a lot more than just this function.

Yes, you are right about that.
https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_analysis.c#L9421-L9424

>DId you already tried to see if it works well.

Well, there changes were quite visible on a test binary I had used. So, I just went with it. 
I have not went down in detail.
![1](https://user-images.githubusercontent.com/29057155/105619409-7b8d0e80-5e18-11eb-832a-87ccde15a08c.png)
![2](https://user-images.githubusercontent.com/29057155/105619411-7e87ff00-5e18-11eb-8faa-4d69c3ce10b5.png)

Maybe, it's better to revert it?

Reply:
> Maybe, it's better to revert it?

No, it seems alright.